### PR TITLE
DEVDOCS-2471: updated placements list

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2471](https://jira.bigcommerce.com/browse/DEVDOCS-2471)

## What changed?
* updated list of placements

### CODE REVIEW

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 1]
```diff
@@ -348,7 +348,7 @@ Widgets are rendered on the storefront as a data tag in the HTML.
 |--|--|
 | Widgets | Widgets are the units of content to be placed on specific pages in a Stencil theme. Each widget is comprised of a widget configuration and a widget template. *There is a limit of `10,000` widgets per store.* |
 | Widget templates | Widget templates are Handlebars-enabled HTML templates which define the widget’s structure on a page. These templates can include conditional logic as well as looping. *There is a limit of `1,000` total custom widget templates per store. This does not include templates pre-provided by BigCommerce.* |
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-overview.md:350]: | Widget templates | Widget templates are Handlebars-enabled HTML templates that define the widget's structure on a page. These templates can include conditional logic as well as looping. *There is a limit of `1,000` total custom widget templates per store. This limit does not include templates pre-provided by BigCommerce.* |

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 2]
```diff
@@ -348,7 +348,7 @@ Widgets are rendered on the storefront as a data tag in the HTML.
...
 | Regions | Regions are specific spots in a Stencil template file where you can place widgets. Regions are defined at the theme file level using the following syntax: `{{{region name="..."}}}`. Many widgets can reside within a given region, and these widgets can have an assigned sort order. |
 | Widget configuration | This is a JSON payload that contains data used when rendering the widget. Each widget has a configuration, and there is a 64kb limit on the size of the JSON. The widget configuration must be valid JSON, but we don’t enforce any additional configuration structure requirements. |
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-overview.md:353]: Regions are specific spots in a Stencil template file where you can place widgets. Regions are defined at the theme file level using the following syntax: `{{{region name="..."}}}`. Many widgets can reside within a given region, and these widgets can have an assigned sort order. |
| Widget configuration | This is a JSON payload that contains data used when rendering the widget. Each widget has a configuration, and there is a 64kb limit on the JSON size. The widget configuration must be valid JSON, but we don’t enforce any additional configuration structure requirements. |

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 3]
```diff
@@ -357,4 +357,4 @@ Widgets are rendered on the storefront as a data tag in the HTML.
 ### Endpoints
 * [Widgets API](/api-reference/storefront/widgets-api)
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-overview.md:358]: * [Widgets API](/api-reference/store-management/widgets)

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 4]
```diff
@@ -357,4 +357,4 @@ Widgets are rendered on the storefront as a data tag in the HTML.
...
 * [Widgets Tutorial](/api-docs/storefront/widgets/widgets-tutorial)
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-overview.md:359]: * [Widgets Tutorial](/api-docs/store-management/widgets/tutorials/tutorial)

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 5]
```diff
@@ -357,4 +357,4 @@ Widgets are rendered on the storefront as a data tag in the HTML.
...
-* [Widgets Code Samples](/api-docs/storefront/widgets/widgets-code-samples)
+* [Widgets Code Samples](/api-docs/storefront/widgets/widgets-code-samples)
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-overview.md:360]: * [Widgets Code Samples](/api-docs/store-management/widgets/samples)

**Review by traci707 — CHANGES_REQUESTED** [review]
I found a few small changes. I also updated the URL for the endpoints. It appears there are some redirects in place.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**